### PR TITLE
PURCHASE-1272 - Replace all Emission Buttons with Palette Button

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/PURCHASE-1272
- Adds `GoogleMapsAPIKey` to Podfile
- Removes `Sale` view which was never shipped

# Bid flow

Screen Shot 2019-07-15 at 2 23 17 PM

Screen Shot 2019-07-15 at 2 14 03 PM

Screen Shot 2019-07-15 at 1 59 18 PM

bidresult_bid_again_after

bidresult_bid_again_after

## New NavigationButton

- Before

auction_results_before

- After

auction_results_after

## Sale 

- Before 

sale_before

- After

sale_after

## Gene 

- Before

refine_before

- After

gene_after

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/__tests__/BillingAddress-tests.tsx [hunk 1]
```diff
@@ -89,12 +89,13 @@ it("updates the validation for country when coming back from the select country
...
 
-  expect(component.root.findByType(Sans).props.children).toEqual("This field is required")
+  expect(component.root.findAllByType(Sans)[0].props.children).toEqual("This field is required")
 
   selectCountry(component, fakeNavigator, billingAddress.country)
 
   // The <Sans12> instances in the BillingAddress screen display error messages
-  expect(component.root.findAllByType(Sans).length).toEqual(0)
+  // FIXME:
+  // expect(component.root.findAllByType(Sans).length).toEqual(0)
```
Comment (ashfurrow) [inline src/lib/Components/Bidding/Screens/__tests__/BillingAddress-tests.tsx]: What's up here?

File: src/lib/Components/Modal.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { Sans } from "@artsy/palette"
+import { Button, Sans } from "@artsy/palette"
+// import { Button } from "@artsy/palette"
```
Comment (ashfurrow) [inline src/lib/Components/Modal.tsx]: Leftover comment here.

File: data/complete.queryMap.json [hunk 3]
```diff
@@ -1,135 +1 @@
...
...    ... on ExternalPartner {\n      id\n    }\n  }\n  images {\n    url\n    aspect_ratio\n  }\n  followedArtists(first: 3) {\n    edges {\n      node {\n        artist {\n          name\n          href\n          slug\n          internalID\n          id\n        }\n      }\n    }\n  }\n  artists {\n    name\n    href\n    slug\n    internalID\n    id\n  }\n}\n\nfragment ShowArtworksPreview_show on Show {\n  id\n  counts {\n    artworks\n  }\n  artworks_connection(first: 6) {\n    edges {\n      node {\n        ...GenericGrid_artworks\n        id\n      }\n    }\n  }\n}\n\nfragment ShowArtistsPreview_show on Show {\n  internalID\n  slug\n  artists {\n    internalID\n    slug\n    href\n    ...ArtistListItem_artist\n    id\n  }\n  artists_without_artworks {\n    internalID\n    slug\n    href\n    ...ArtistListItem_artist\n    id\n  }\n}\n\nfragment Shows_show on Show {\n  nearbyShows(first: 20) {\n    edges {\n      node {\n        id\n        ...ShowItem_show\n      }\n    }\n  }\n}\n\nfragment LocationMap_location on Location {\n  id\n  internalID\n  city\n  address\n  address_2\n  postal_code\n  summary\n  coordinates {\n    lat\n    lng\n  }\n  day_schedules {\n    start_time\n
```
Comment (ashfurrow) [inline data/complete.queryMap.json]: Can we undo this?

Comment (kierangillen) [reply data/complete.queryMap.json]: Updated this!

File: src/lib/Components/Modal.tsx (hunk 2 above)
Comment (kierangillen) [reply src/lib/Components/Modal.tsx]: Thanks cleaned this up.

File: src/lib/Components/Bidding/Screens/__tests__/BillingAddress-tests.tsx (hunk 1 above)
Comment (kierangillen) [reply src/lib/Components/Bidding/Screens/__tests__/BillingAddress-tests.tsx]: Fixed this. Now that I replaced some existing buttons with the new palette `Button` the rendered element tree has changed and now there are additional `Sans` elements than previously were rendering.

**Pull request merged**